docs/book/helpers/escape.md

```diff
@@ -14,6 +14,7 @@ More information to the operation and the background of security can be found
 in the
 [documentation of laminas-escaper](https://docs.laminas.dev/laminas-escaper/configuration/).
 
+<!-- markdownlint-disable-next-line header-increment -->
```

Comment:
It's okay for this pull request.

---

A solution for the future would be to find a headline before this section or to find a better markup than a blockquote.

> The blockquote element represents a section that is quoted from another source.

https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element